plaidml/bridge/openvino/src/plaidml_plugin/ops/sigmoid.cpp

```diff
@@ -0,0 +1,24 @@
+/ Copyright(C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset1.hpp"
+
+#include "plaidml/op/op.h"
+
+    using namespace plaidml;      // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace PlaidMLPlugin {
+
+static OpRegistration reg("sigmoid", [](const Context& ctx) {
+  auto* layer = dynamic_cast<ngraph::opset1::Sigmoid*>(ctx.layer);
```

Comment:
Don't include the creation of a `layer` variable if you never need to use it (e.g. on ops with no attributes). It causes compiler errors due to the unused variable.

Reply:
I see. It usually gives a warning though.
I tried removing it, but the errors persist :(

Reply:
Yes, we have warnings as errors turned on, so the warning becomes an error.

Hmm, the same error persists? And you fixed the `/` -> `//` comment above?